DOCKER/docker-compose

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+
+services:
+
+  postgis:
+    build:
+      context: .
+      dockerfile: dockerfile-toms-test-postgis
+    hostname: toms_postgis2
+    environment:
+      - POSTGRES_PASSWORD=postgis
+      - POSTGRES_DBNAME=gis
+    volumes:
+      - "pg_data_toms_test:/var/lib/postgresql"
+      - /home/QGIS/plugins/TOMs/DATAMODEL:/docker-entrypoint-initdb.d
+      - /home/QGIS/plugins/TOMs/test/data:/docker-entrypoint-initdb.d
+    ports:
+      - 5435:5432
+
+  qgisserver:
+    image: openquake/qgis-server:ltr
+    hostname: qgis-server1
+    environment:
+      - PGSERVICEFILE=/io/pg_service/.pg_service.conf
+    volumes:
+      # Wherever you want to mount your data from
+      - /home/tim/qgis_server/data:/io/data
+      - /home/QGIS/plugins:/io/plugins
+      - /home/QGIS/resources:/var/lib/qgis/.local/share/QGIS/QGIS3/profiles/default/svg
```

Comment:
Use relative paths here too, to use the QGIS project file and plugin from the repo directly.

It may be cleaner at some point to move the TOMs plugin to a subfolder of your git repo (it's generally a good practice, as it helps keep the plugin code clean from config files or other unrelated files).

Reply:
Does this mean that the repository structure becomes:

TOMs
|- DATAMODEL
|- QGIS
|- test
    |- data
    |- expected_results
|- TOMs (or perhaps another name)
    |- CadNodeTool
    |- InstantPrint
    |- help
    ...

Reply:
Yes maybe you can call it "TOMs_plugin" so it's even more clear what it's about.
You'd also have to move everything that's used by the plugin in the folder too (e.g. CadNodeTool).